txnkv/txn.go

```diff
@@ -264,16 +262,19 @@ func (txn *Transaction) Rollback() error {
 	if !txn.valid {
 		return kv.ErrInvalidTxn
 	}
+	start := time.Now()
+	defer func() { metrics.TxnCmdHistogram.WithLabelValues("rollback").Observe(time.Since(start).Seconds()) }()
```

Comment:
em, why not combine them into one defer fun?

Reply:
good idea